src/components/SignUpForm/styles.scss

```
@@ -0,0 +1,37 @@
+@import "../../styles/main.scss";
+
+.container {
+    min-height: 100vh;
+    @include flex-center();
+}
+
+.signUp-form {
+    margin: auto;
+    height: rem(400px);
+    width: rem(650px); 
+    flex-direction: column;
+    
+    background-color: $background;
+  
+    @include flex-center();
```

```suggestion
    background-color: $background;
    @include flex-center();
```
```suggestion
    
    background-color: $background;
  
    @include flex-center();
```
Please remove some extra spaces.